augly/image/functional.py

```diff
@@ -1786,3 +1786,56 @@ def vflip(
     imutils.get_metadata(metadata=metadata, function_name="vflip", **func_kwargs)
 
     return imutils.ret_and_save_image(aug_image, output_path)
+
+
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    level: float = 5.0,
+    max_level: float = 10.0,
```

Comment:
Did you test running the augmentation with these args?

Reply:
Actually, I want to do some changes basically in `level` because when I was running it is mostly completely skewing the image. so level should 1.0. I'll make changes too.

Reply:
When I applied level : 1.0 and Max_level : 10.0 I'll put these by default.

![test_Skew](https://user-images.githubusercontent.com/56601466/123136895-109c3e00-d471-11eb-85fc-461c2e97e63e.png)

Reply:
Sure, this looks fine! Please make this change in the code above then (i.e. change 5.0 to 1.0 in line 1794).

Reply:
Yes, I've made changes in all the files that have level value.

Reply:
I don't see any changes. The change you need to make is right here, on line 1794.